docs/platform/howto/access-suspended-due.rst-to-past-due-invoice

```diff
@@ -0,0 +1,24 @@
+Access Suspended Due to Past Due Invoice
+===================================================================
+
+How to reactivate suspended account.
```

Comment:
```suggestion
Reactivate a suspended account
```

Reply:
Is it the account or the project that is suspended?

Reply:
it is a project, in my case there was only 1 project so I assume we can change it to project